lib/components/upload_form.dart

```diff
@@ -906,62 +932,95 @@ class _UploadFormState extends State<UploadForm> {
+                                                  task.status ==
+                                                      UploadStatus
+                                                          .complete)) ...[
+                                            Flexible(
+                                              flex: 2,
+                                              child: ArDriveProgressBar(
+                                                height: 4,
+                                                indicatorColor: task.status ==
+                                                        UploadStatus.failed
+                                                    ? ArDriveTheme.of(context)
+                                                        .themeData
+                                                        .colors
+                                                        .themeErrorDefault
+                                                    : task.progress == 1
+                                                        ? ArDriveTheme.of(
+                                                                context)
+                                                            .themeData
+                                                            .colors
+                                                            .themeSuccessDefault
+                                                        : ArDriveTheme.of(
+                                                                context)
+                                                            .themeData
+                                                            .colors
+                                                            .themeFgDefault,
+                                                percentage: task.progress,
+                                              ),
+                                            ),
+                                            Flexible(
+                                              child: Text(
+                                                '${(task.progress * 100).toInt()}%',
+                                                style: ArDriveTypography.body
+                                                    .buttonNormalBold(
+                                                  color:
+                                                      ArDriveTheme.of(context)
                                                           .themeData
                                                           .colors
-                                                          .themeSuccessDefault
-                                                      : ArDriveTheme.of(context)
+                                                          .themeFgDefault,
+                                                ),
+                                              ),
+                                            ),
+                                          ],
```

Comment:
Even tho we don't have the progress, we can render a `LoadingAnimationWidget.prograssiveDots` widget to give some visual feedback.
```suggestion
                                          ], else LoadingAnimationWidget.prograssiveDots(...),
```

Reply:
That makes sense!

Reply:
Applied: 39c5b40c1177d4ba9b0580ef13405dbd8c40277c